avm-transpiler/src/transpile.rs

```diff
@@ -1313,8 +1280,6 @@ pub fn map_brillig_pcs_to_avm_pcs(brillig_bytecode: &[BrilligOpcode<FieldElement
     pc_map[0] = 0; // first PC is always 0 as there are no instructions inserted by AVM at start
     for i in 0..brillig_bytecode.len() - 1 {
         let num_avm_instrs_for_this_brillig_instr = match &brillig_bytecode[i] {
-            BrilligOpcode::Cast { bit_size: BitSize::Integer(IntegerBitSize::U1), .. } => 2,
-            BrilligOpcode::Not { bit_size: IntegerBitSize::U1, .. } => 3,
             _ => 1,
         };
```

Comment:
We can actually remove this entire function and remove its usage from the transpiler because all opcodes are now a 1:1 mapping between brillig and avm :tada:

I'll do so in a followup PR

Reply:
Ok but remember that we are moving to byte-indexed PC so we will have to write a much more complicated one soon ;)

Reply:
Ah right.... In that case, I'll leave this function and the other functions (to patch debug info and asserts) since we'll need something similar after the change to byte-indexed PC.